integration/spark/src/main/spark3/java/io/openlineage/spark3/agent/lifecycle/plan/DataSourceV2RelationVisitor.java

```diff
@@ -106,16 +127,52 @@ private Provider findDatasetProvider(LogicalPlan plan) {
             .build());
   }
 
+  private OpenLineage.Dataset findDatasetForAzureCosmos(DataSourceV2Relation relation) {
+    String namespace = "azurecosmos";
+    String relationName = relation.table().name().replace("com.azure.cosmos.spark.items.", "");
+    int expectedParts = 3;
+    String[] tableParts = relationName.split("\\.", expectedParts);
+    String tableName;
+    if (tableParts.length != expectedParts) {
+      tableName = relationName;
+    } else {
+      tableName =
+          String.format(
+              "https://%s.documents.azure.com/dbs/%s/colls/%s",
```

Comment:
Is `https://serviceName.documents.azure.com/dbs/databaseName/colls/collectionName` link to internal interface, is used in Azure as a dataset/collection/table name, or is something that would be more recognizable to Cosmos user than for example `azurecosmos://serviceName.databaseName.collectionName`? 

If not, then maybe just use `serviceName.databaseName.collectionName` as a name - consistently with other databases, like bigquery, redshift, snowflake.

Reply:
@mobuchowski Thank you for the review!  That URI is the actual endpoint for the "collection" (i.e. table) of the Cosmos Db. You can actually use that URL as part of the [REST API](https://docs.microsoft.com/en-us/rest/api/cosmos-db/get-a-collection) to get that specific collection's metadata.

For anyone using the Cosmos DB REST API (or subsequently Azure Purview), they would be familiar with the pattern provided.  Does that help?

Reply:
Yes, thanks 👍